test/built-ins/RegExp/prototype/Symbol.matchAll/isregexp-called-once.js

```diff
@@ -31,7 +30,8 @@ var o = {
   get [Symbol.match]() {
     ++count;
     return false;
-  }
+  },
+  flags: "",
```

Comment:
should this be `get flags() { }` and increment a `flagsCount`, to ensure this is looked up?

Reply:
this can be added as a follow up if you don't mind.

Reply:
certainly